interface/src/avatar/MyCharacterController.cpp

```diff
@@ -291,10 +291,12 @@ void MyCharacterController::initRayShotgun(const btCollisionWorld* world) {
             for (int32_t j = 0; j < maxJ; ++j) {
                 btVector3 localRayEnd(offsetX + (btScalar)(j) * stepX, divisionLine + (btScalar)(i) * stepY, 0.0f);
                 btVector3 localRayStart = localRayEnd - reach;
-                MeOnlyResultCallback result(_rigidBody);
-                world->rayTest(position + rotation * localRayStart, position + rotation * localRayEnd, result);
-                if (result.m_closestHitFraction < 1.0f) {
-                    _topPoints.push_back(localRayStart + result.m_closestHitFraction * reach);
+                if (localRayStart != localRayEnd) {
```

Comment:
Did you consider inverting the logic here?  
I mean going like:
```
if(start == end)  
    continue;
```
I think it'd result in less changes to the previous code + be less indented, which I think is a good thing here.